app/src/main/java/com/example/myapplication/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.myapplication.mvp.model
+
+class CalculatorModel{
+    private var operator1 : String = ""
```

Comment:
Please, put this strings into a StringsUtils class or in a resources .

Reply:
also, try to avoid digits in variable names.
operatorOne seems better

Reply:
done